MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -401,6 +401,11 @@
 				shaderIntFuncsFeatures->shaderIntegerFunctions2 = true;
 				break;
 			}
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_ATOMIC_FLOAT_FEATURES_EXT: {
+				auto* atomicFloatFeatures = (VkPhysicalDeviceShaderAtomicFloatFeaturesEXT*)next;
+				atomicFloatFeatures->shaderBufferFloat32AtomicAdd = _metalFeatures.mslVersion >= 030000;
```

Comment:
What about the other atomic float capabilities? From the Metal docs, I would expect that all four buffer 32-bit  float capabilities are available:

 ```
shaderBufferFloat32Atomics
shaderBufferFloat32AtomicAdd
shaderSharedFloat32Atomics
shaderSharedFloat32AtomicAdd
```

Metal supports supports the load, store, and exchange of `shaderBufferFloat32Atomics`. And the `Shared` capabilities seem to be covered by Metal's `threadgroup A* object` function parameter types.

Am I missing something? Or did you exclude them due to time constraints relative to your requirements?

Reply:
Ok I went ahead and enabled those other flags, and addressed the other comments above in the latest push.  Thanks!